src/index.ts

```diff
@@ -377,36 +360,31 @@ export async function install(
     }
 
     fs.writeFileSync(
-      path.join(destLoc, 'import-map.json'),
+      path.join(config.options.dest, 'import-map.json'),
       JSON.stringify({imports: importMap}, undefined, 2),
       {encoding: 'utf8'},
     );
   }
   Object.entries(assetObject).forEach(([assetName, assetLoc]) => {
-    mkdirp.sync(path.dirname(`${destLoc}/${assetName}`));
-    fs.copyFileSync(assetLoc, `${destLoc}/${assetName}`);
+    mkdirp.sync(path.dirname(`${config.options.dest}/${assetName}`));
+    fs.copyFileSync(assetLoc, `${config.options.dest}/${assetName}`);
   });
   return true;
 }
 
 export async function cli(args: string[]) {
-  const {
-    help,
-    sourceMap,
-    babel = false,
-    exclude = ['**/__tests__/*', '**/*.@(spec|test).@(js|mjs)'],
-    include,
-    nomodule,
-    nomoduleOutput = 'app.nomodule.js',
-    optimize = false,
-    strict = false,
-    clean = false,
-    dest = 'web_modules',
-    externalPackage: externalPackages = [],
-  } = yargs(args, {array: ['externalPackage']});
-  const destLoc = path.resolve(cwd, dest);
+  // Load config
+  const cliFlags = yargs(args, {array: ['exclude', 'externalPackage']});
+  const {config, errors} = await loadConfig({options: cliFlags as SnowpackConfig['options']});
+
+  if (Array.isArray(errors) && errors.length) {
+    errors.forEach(logError);
+    process.exit(0);
+  }
+
+  config.options.dest = path.resolve(cwd, config.options.dest);
```

Comment:
This is the same line as `const destLoc = path.resolve(cwd, dest);`. To the comment above, I thought it was cleaner to use 1 consistent naming pattern in both `cli()` and `install()`, rather than having to grok things like “Is `babel` the same as `isBabel`? Is `dest` the same as `destLoc`?” etc.

But that said, yeah it does seem weird that we’re re-assigning a variable here. I’ll add a comment.

Reply:
ah, gotcha. What's changed now is that this went from a top-level var to a property of `config.options`, an object created in `config.ts`. So now it feels a bit strange to modify this config object outside of `config.ts`. 

If you can move this line into config.ts to be with the others, that would be great. But if that's more trouble than its worth, moving it up to the line immediately following `const config = ... ` above would also be fine.